packages/contracts/test/20_integration-testing/test-helpers.ts

```diff
@@ -264,8 +264,8 @@ export async function updateFromBuildTest(
     .connect(deployer)
     .grant(dao.address, psp.address, DAO_PERMISSIONS.ROOT_PERMISSION_ID);
 
-  // Install build 1.
-  const pluginSetupRefBuild1 = {
+  // Install previous plugin build (build previous to the latest one)
```

Comment:
```suggestion
  // Install a previous build with build number `build`
```
Is this better? Feel free to ignore this if you disagree.